mgmt/LocalManager.h

```diff
@@ -94,7 +94,7 @@ class LocalManager : public BaseManager
   void processDrain(int to_drain = 1);
   void rollLogFiles();
   void clearStats(const char *name = nullptr);
-  void hostStatusSetDown(const char *name);
+  void hostStatusSetDown(const char *req, int len);
```

Comment:
It's still len here ?

Reply:
Should arguments be renamed to indicate it is marshaled data?